File: Corredor/Digitales/Java/src/Estudiante.java
public class Estudiante {
    private String nombre;
    private String instrumento;
    private int nivel;

    public Estudiante(String nombre, String instrumento, int nivel) {
        this.nombre = nombre;
        this.instrumento = instrumento;
        this.nivel = nivel;
    }

    public void practicar() {
        System.out.println(nombre + " practica " + instrumento + " (nivel " + nivel + ").");
    }
}

File: Corredor/Digitales/Go/estudiante.go
package main

import "fmt"

type Estudiante struct {
	nombre      string
	instrumento string
	nivel       int
}

func NuevoEstudiante(nombre, instrumento string, nivel int) Estudiante {
	return Estudiante{nombre, instrumento, nivel}
}

func (e Estudiante) Practicar() {
	fmt.Printf("%s practica %s (nivel %d).\n", e.nombre, e.instrumento, e.nivel)
}